src/Umbraco.Web/PublishedCache/NuCache/ContentNode.cs

```diff
@@ -149,7 +150,7 @@ private IPublishedContent GetModel(ref IPublishedContent model, ContentData cont
             // more than 1 instance, but the lock below ensures we only ever return
             // 1 unique instance - and locking is a nice explicit way to ensure this
 
-            var m = new PublishedContent(this, contentData, _publishedSnapshotAccessor, _variationContextAccessor, _publishedModelFactory).CreateModel(_publishedModelFactory);
+            var m = new PublishedContent(this, contentData, _publishedSnapshotAccessor, _variationContextAccessor, _publishedModelFactory, null).CreateModel(_publishedModelFactory);
```

Comment:
I set this to `null` for now, because I think the better way here is to inject `PublishedContext` from DI instead of newing up here